src/InputToolbar.js

```diff
@@ -33,15 +33,19 @@ export default class InputToolbar extends React.Component {
   }
 
   keyboardWillShow() {
-    this.setState({
-      position: 'relative',
-    });
+    if (this.state !== 'relative') {
```

Comment:
Should this be `this.state.position`? Same for the next if condition also.

Reply:
This is probably what caused this bug: https://github.com/FaridSafi/react-native-gifted-chat/pull/705#pullrequestreview-120090347